lib/querystring.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+const {
+  StorageObject,
+  percentEscape,
+  unescapeBuffer
+} = require('internal/querystring');
 const QueryString = module.exports = {
   unescapeBuffer,
```

Comment:
Not related to deprecations or anything, just now that we are refactoring things, might as well :P